src/rust/ide/lib/view/src/debug_scenes/documentation_generator_view.rs

```diff
@@ -0,0 +1,80 @@
+#![allow(missing_docs)]
+
+use ensogl::traits::*;
+use ensogl::prelude::*;
+
+use ensogl::system::web;
+use ensogl::system::web::NodeInserter;
+use ensogl::display::symbol::DomSymbol;
+use web::StyleSetter;
+use ensogl::display::world::*;
+use ensogl::display::navigation::navigator::Navigator;
+
+use nalgebra::Vector2;
+use nalgebra::Vector3;
```

Comment:
again, unused imports. These are part of prelude. Something is really bad with linter here. Please check why its not working properly.